app/views/conferences/_details.html.slim

```diff
@@ -1,24 +1,12 @@
 .conference
   dl.dl-horizontal
     dt Location
-    dd = @conference.location
+    dd #{@conference.city}, #{@conference.country}
     dt Date
     dd = @conference.date_range
     dt Twitter
     dd = format_conference_twitter(@conference.twitter)
     dt Homepage
     dd = link_to @conference.url, @conference.url
-    dt Tickets total
```

Comment:
Just for comprehending the feature: we don't work with this data anymore? If not, should we remove the corresponding database columns as well?